reflex-algos/src/main/python/mlops/parallelm/mlops/time_capture/parsers.py

```diff
@@ -175,23 +176,16 @@ def _parse_mlapp(self, file_path):
         """
         with open(file_path, 'r') as f:
             reader1 = f.read()
-        reader1 = reader1.replace('wfNodes', ',wfNodes')
-        reader1 = reader1.replace('GroupID', ',GroupID')
-        reader1 = reader1.replace('Mode', ',Mode')
-        reader1 = reader1.replace('Type', ',Type')
-        reader1 = reader1.replace(',', '","')
-        reader1 = reader1.replace(':', '":"')
-        reader1 = reader1.replace('" [', '[{"')
-        reader1 = reader1.replace(']"', '"}]')
-        reader1 = reader1.replace(' ', '')
-        reader1 = reader1.replace(',"PipelineID', '},{"PipelineID')
-        nd1 = '{"' + reader1 + '"}'
-        kd = json.loads(nd1)
+        kd = json.loads(reader1)
         self._mlapp_id = kd["WorkflowID"]
         self._model_policy = kd["modelPolicy"]
-        self._nodes_number = len(kd["wfNodes"])
-        self._nodes_id = [nodes["PipelineID"] for nodes in kd["wfNodes"]]
-        self._nodes_type = [nodes["Type"] for nodes in kd["wfNodes"]]
+        self._nodes_number = len(kd[IONJsonConstants.PIPELINE_INSTANCES_SECTION])
+
+        self._nodes_id1 = [nodes[IONJsonConstants.PIPELINE_INSTANCE_ID_TAG] for nodes in kd[IONJsonConstants.PIPELINE_INSTANCES_SECTION]]
```

Comment:
i dont see this being used anywhere in this review - do we still need this variable? if so, please change its name?

Reply:
remove